client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -0,0 +1,26 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormattedHeader from 'components/formatted-header';
+
+class JetpackOnboardingBusinessAddressStep extends React.Component {
+	render() {
```

Comment:
Let's make it either stateless/functional our a `PureComponent`.

In principle, I like to start with stateless whenever possible. We're arguable going to need state or event handlers in most of these steps, but usually, it's easier to forget to turn something that doesn't back into a stateless one.

`PureComponent` could be a good compromise.

Reply:
Done 👍 8547b85e6cade12ff5ea0a70b10c94b40bcfce8a